.circleci/config.yml

```diff
@@ -259,7 +267,7 @@ jobs:
             python -m venv env
             . env/bin/activate
             pip install dimod --no-index -f dist/ --force-reinstall --no-deps
-            pip install << parameters.dependencies >> --upgrade --only-binary=numpy
+            pip install "<< parameters.dependencies >>" --upgrade --only-binary=numpy
```

Comment:
I recommend quoting individual specifiers -- below in the list of parameter values -- not the parameter value here. That way you can use multiple specifiers per parameter (e.g. `"'numpy<2' scipy~=1.0"`). For example, see cloud-client's config, or [dwave-inspector's](https://github.com/dwavesystems/dwave-inspector/blob/master/.circleci/config.yml) one.

Alternatively, you can use tilde in version specifier, and avoid quoting altogether.

Reply:
I will likely revert this commit as part of the NumPy 2.0 PR, so I will leave it be for now.